cartoframes/datasets/dataset.py

```diff
@@ -102,6 +102,15 @@ def set_geodataframe(self, gdf):
     def get_table_name(self):
         return self._table_name
 
+    def get_context(self):
+        return self._cc
```

Comment:
I made a comment here on internal naming of this context: https://github.com/CartoDB/cartoframes/pull/717#discussion_r290783954

Reply:
Also, what about setting the context as a `@property` / public attribute instead of these `get_`/`set_` methods?

Reply:
It is a different case, but for sure, we should have used `context` instead of `cc` in Dataset. But I prefer to change `cc` by `context` in a specific task

Reply:
Ok, we could do it